docs/index.md

```diff
@@ -1,8 +1,12 @@
 # Pebble
 
-_Take control of your internal daemons!_
+**Pebble** is a lightweight Linux service manager.
 
-**Pebble** helps you to orchestrate a set of local service processes as an organised set. It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that it can easily manage non-system processes independently from the system services, but it was designed with unique features that help with more specific use cases.
+It helps you orchestrate a set of local processes as an organised set. It resembles well-known tools such as _supervisord_, _runit_, or _s6_, in that it can easily manage non-system processes independently from the system services. However, it was designed with unique features such as layered configuration and an HTTP API that help with more specific use cases.
+
+Pebble fulfils the need for streamlined, dependable, and secure service management. It empowers you to efficiently operate your services with ease, resolve service interdependencies, configure tailored health checks to automatically restart services upon failure, and gain insightful visibility into Pebble server events for a comprehensive understanding of operational dynamics. Pebble also supports simple identity and access control, ensuring secure usage in shared environments.
+
+Pebble caters to a diverse user base managing services both locally and within containerised environments, offering a robust solution for your workload management needs. It is also extensively used alongside [Juju and Charms](https://juju.is/).
```

Comment:
Again lots of jargon here: "caters", "diverse", "robust solution", "X needs". Open to better suggestions, but perhaps:

```suggestion
Pebble is useful for developers who are building [Juju charms on Kubernetes](https://juju.is/docs/sdk/from-zero-to-hero-write-your-first-kubernetes-charm), creating [Rocks](https://documentation.ubuntu.com/rockcraft/en/latest/explanation/rocks/) or Docker images, or orchestrating services in the virtual machine.
```

Reply:
The suggested edit is committed, it says the same thing with plain words and I like it.